recipes-core/images/mistysom-image.bb

```diff
@@ -1,6 +1,6 @@
 SUMMARY = "MistySOM image."
 
-IMAGE_FEATURES += "splash package-management ssh-server-dropbear hwcodecs"
+IMAGE_FEATURES += "splash package-management ssh-server-openssh hwcodecs"
```

Comment:
yeSSSSS! I've had this on my side for a while but never got into a PR.... Thankssssssssssssss!